Lombiq.HelpfulLibraries.OrchardCore/Users/AuthorizationServiceExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Http;
+using OrchardCore.Security.Permissions;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Authorization;
+
+public static class AuthorizationServiceExtensions
+{
+    public static Task<bool> AuthorizeCurrentUserAsync(
+        this IAuthorizationService service,
+        HttpContext context,
+        Permission permission) =>
+        context?.User is { Identity.IsAuthenticated: true } user
+            ? service.AuthorizeAsync(user, permission)
```

Comment:
Is it desired for this invocation to throw an exception when `permission` is `null`? Because that's what will happen.

Reply:
I think so. Typically the permission of an authorization check is known at compile time. So it being null is definitely wrong.